brave/brave-core
# fix news feed crash on reaching last item in android

Resolves https://github.com/brave/brave-browser/issues/25312

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open the app
2. Enable the Brave News
3. Scroll down to last item of news feed (Select few news sources so you can reach bottom of the feed)
4. App should not crash
